### ISSUE TITLE
iframe on Support home page has invalid `aria-label`

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4043]
## Area or Component
simplereport.gov/support - the embedded YouTube player

## Quick Summary
The iframe for the YouTube player is embedded in a `div` element with an `aria-label` field, but `aria-label` is not supported on `div` elements.

## Accessibility Violation
Elements must only use allowed ARIA attributes: https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr?application=AxeChrome

## Action Requested
Remove the aria-label from the div (it may be allowed on the iframe element)

## Screenshots (optional)
![Screen Shot 2022-07-21 at 6 41 51 PM](https://user-images.githubusercontent.com/80282552/180343359-008d7cd1-bcbd-4088-b8c6-7a15ae59e504.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=1832428693&range=15:15

**Comment by georgehager** [comment]
@emmastephenson @emyl3 Is it possible this issue has already been resolved? I am looking at simplereport.gov/support and inspecting the element captured in the screenshot above, and it is structured differently:
![Screen Shot 2022-08-18 at 8 55 06 AM](https://user-images.githubusercontent.com/89797785/185400329-b7769946-bc5f-4c81-82a6-76a55954daeb.png)

I see this structure locally as well.

The enclosing div only has a class (no offending aria-label), and the iframe itself has a title, which is interpreted by screen readers:
> [If you give your <iframe> a title... aria-label is not necessary. But, if present, the aria-label will take precedence over the title...as your iframe's...accessible name....](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)

Not sure how to proceed with this, or if it's possible I'm looking in the wrong location?

**Comment by georgehager** [comment]
Upon further investigation of git history, I can't find a time that `aria-label` was applied to a `div` enclosing an `i-frame`... 🤔

**Comment by emyl3** [comment]
@georgehager the warning was going off on a `<div>` inside the youtube `<iframe>` itself so you are right in that I don't think there is much we can do here...

<img width="1233" alt="Screen Shot 2022-08-18 at 9 12 45 AM" src="https://user-images.githubusercontent.com/20211771/185417266-c941b591-1d05-40c5-b97e-9d315715ff8b.png">

**Comment by georgehager** [comment]
@emyl3 thank you! my bad, I didn't realize it was _inside_ the `iframe` itself. that's all autogenerated thought, right? I'm not finding a way to turn off this "feature" 😬

**Comment by emyl3** [comment]
Yes, it is 😢 Yeah, I believe there is nothing we can do about that inner div coming from the iframe 😮‍💨

**Comment by georgehager** [comment]
@emyl3 Poking around to see if this is a known problem, and it appears it's been around for a while and as recently as 2021 [it was still causing problems](https://adrianroselli.com/2017/08/inaccessible-youtube-embeds.html#comment-216891). Looks like it's specifically an issue with Google/Youtube. It seems like the only solution is writing a script to remove it, do we think that is a viable solution? Otherwise, we've implemented the `iframe` as widely [recommended](https://dequeuniversity.com/tips/provide-iframe-titles).

**Comment by georgehager** [comment]
I also posted in the [google accessible](https://groups.google.com/g/accessible) group to see if there's any more information on this.

**Comment by emmastephenson** [comment]
Thanks for tracking this down George! Much like with #4109, it sounds like there's not much we can do here 😞 The Deque guidelines do acknowledge that for iframes in particular you don't always have control over the content, so I think we're ok to leave this as-is - I don't know that it's worth the trouble to try and override this with a script when it's an issue at the YouTube level.
If you hear anything back from the Google accessibility folks we can make a note of it here, otherwise I think it's ok to close this ticket.

**Comment by georgehager** [comment]
Thanks, @emmastephenson! I agree with you that scripting is not a reasonable solution in this context. Closing now, since it may be some time before I hear back. Will make a note if I do end up hearing anything!
